### ISSUE TITLE
Does disabling FSR downscaling also disable Dynamic Resolution

### ISSUE THREAD

**Issue by Pandorify** [HolographicWings/TOTK-Mods-collection#134]
I'm wondering if the disable FSR downscaling mod also disables Dynamic Resolution. It's not very clear from the docs and I don't see any standalone mods to remove Dynamic Resolution. Thanks

**Comment by SpockBauru** [comment]
Disable Dynamic Resolution was taken off because was causing too many issues for lots of people.
The closest thing we have now is the Disable Quality Reduction mod, which needs more testing. Please try this one and share your findings :D

**Comment by Pandorify** [comment]
Is there anywhere else I can get the mod which has been ported to 1.2.0?

**Comment by Ryusennin** [comment]
The original Disable Quality Reduction mod is part of Chuck's visual patches for 1.1.2:
https://www.reddit.com/user/ChucksFeedAndSeed/comments/13sofgg/totk_v112_visual_patches_chuckpatch_fsr_disable/

**Comment by hoverbike1** [comment]
> Is there anywhere else I can get the mod which has been ported to 1.2.0?

Yes, we've recently updated the archive to feature 1.2.0 mods, most of them so far, at least.
Youll want to enable "Disable Quality Reduction", found in the "graphics" folder.
This patch not only prevents the game from swapping model LODs when slowdowns occur, but also prevents it from reducing the resolution!
